cypress/Shared/CreateMeasurePage.ts

```diff
@@ -17,11 +17,11 @@ export type CreateMeasureOptions = {
     elmJson?: string,
     measureNumber?: number,
     measureScoring?: string,
-    patientBasis?: boolean,
-    twoMeasures?: boolean,
```

Comment:
From what I have learned recently, the option `twoMeasures` should be removed/retired and replaced with `measureNumber` functionality.
I included that change here.